src/internal/nativeInterface.ts

```diff
@@ -0,0 +1,25 @@
+import { Platform, NativeModules, PlatformOSType } from 'react-native';
+import { DeviceInfoNativeModule } from './privateTypes';
+
+let RNDeviceInfo: DeviceInfoNativeModule | undefined = NativeModules.RNDeviceInfo;
+
+const OS: PlatformOSType = Platform.OS;
+
+// @ts-ignore
+if (OS === 'web' || OS === 'dom') {
```

Comment:
not saying this is wrong, but after a year I just finally had to introduce my first ts-ignore in my work project. Is there nothing worth it (from a cost/benefit perspective) that can be done to type these for real?

Reply:
I felt dirty adding this one.  `web` is typed part of `PlatformOSType`, which sadly doesn't include `dom`.  I also wasn't sure if `dom` was actually a valid value?  Perhaps something that was added as part of the cover everything `web` related.  

We could extend/union the current type and add `dom`, however I first wanted to understand where `dom` actually came from.

Reply:
I believe I added dom so that it would work with jest as well, maybe there's a typing in there

Reply:
Oh great, I'll dig into that.

Reply:
don't bother investigating I just checked and couldn't see it https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/jest I might be missing something transitive but for now I'd merge and we can fix later